service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/dataproccluster/CreateDataprocClusterStep.java

```diff
@@ -221,13 +222,14 @@ static Cluster setFields(
 
     // Set initialization script
     // TODO PF-2828: Add WSM default post-startup script
-    if (!StringUtils.isEmpty(creationParameters.getInitializationScript())) {
+    List<String> initializationScripts = creationParameters.getInitializationScripts();
+    if (initializationScripts != null) {
```

Comment:
Does this need to handle an empty list being passed in? As-is this will pass an empty list to the config object, which seems different than the previous behavior of leaving it unset

Reply:
Good catch. Passing in an empty list is OK and doesn't cause issues.